src/Cake.Common.Tests/Unit/Tools/VSTest/VSTestRunnerTests.cs

```diff
@@ -204,7 +204,7 @@ public void Should_Use_SettingsFile_If_Provided()
             var result = fixture.Run();
 
             // Then
-            Assert.Equal("\"/Working/Test1.dll\" /Settings:Local.RunSettings", result.Args);
+            Assert.Equal("\"/Working/Test1.dll\" /Settings:\"/Working/Local.RunSettings\"", result.Args);
```

Comment:
Absolutely. Got two questions:
 1. Does this mean I have to add a `Name with spaces.RunSettings` file to the project? Where is the `Local.RunSettings` file? Is VSTest actually running against a real file in this test or are you just checking the arguments string?
 2. Should a test be added specifically for whitespace in *every* parameter or is it not worth it? I'm asking because I'm about to add some more after this PR.